### ISSUE TITLE
Compatibilty with vue-meteor-tracker?

### ISSUE THREAD

**Issue by harry-73** [jamauro/pub-sub#5]
Hi,

I am trying to use this package with  https://github.com/meteor-vue/vue-meteor-tracker.

I defined in the client  `main.js` file:

```
import { PubSub } from 'meteor/jam:pub-sub';
PubSub.configure({
  cache: true, // defaults to false
  cacheDuration: 30 // defaults to 30 seconds
});
```

I subscribe to a publication and when I leave the component, all entries in minimongo for the given collection are deleted without any delay. 

Thanks.

**Comment by jamauro** [comment]
What does your `.meteor/packages` file look like?

Can you try putting `jam:pub-sub` above `vite-bundler` if it's not already?

**Comment by harry-73** [comment]
I put `jam:pub-sub` above. Same result.

My `packages` file:
```
# Meteor packages used by this project, one per line.
# Check this file (and the other files in this directory) into your repository.
#
# 'meteor add' and 'meteor remove' will edit this file for you,
# but you can also edit it by hand.

meteor-base@1.5.2-beta300.6             # Packages every Meteor app needs to have
mongo@2.0.0-beta300.6                   # The database Meteor supports right now
reactive-var@1.0.13-beta300.6            # Reactive variable for tracker

standard-minifier-css@1.9.3-beta300.6   # CSS minifier run for production mode
              # ECMAScript 5 compatibility for older browsers
ecmascript@0.16.8-beta300.6              # Enable ECMAScript2015+ syntax in app code
           # Enable TypeScript syntax in .ts and .tsx modules
shell-server@0.6.0-beta300.6            # Server-side component of the `meteor shell` command

tracker@1.3.3-beta300.6                 # Dependency tracker to allow reactive callbacks
static-html@1.3.3-beta300.6             # Define static page content in .html files

accounts-password@3.0.0-beta300.6
momentjs:moment
autoupdate@2.0.0-beta300.6
mdg:validated-method
ddp-rate-limiter@1.2.1-beta300.6
audit-argument-checks
ejson@1.1.4-beta300.6
matb33:collection-hooks
lamhieu:hypersubs
dynamic-import@0.7.4-beta300.6
browser-policy@1.1.3-beta300.6
jam:pub-sub
jorgenvatle:vite-bundler@2.0.0-beta.14
http
jkuester:http
meteortesting:mocha@3.1.0-beta300.0
standard-minifier-js@3.0.0-beta300.6
useraccounts:core
useraccounts:unstyled
```

**Comment by jamauro** [comment]
Ok thanks. I'll set aside some time this weekend to try and diagnose the issue.

**Comment by jamauro** [comment]
I just published a fix in `v0.2.2`. To test it locally, I put this in `/imports/ui/main.js`, aka the `clientEntry` point defined in the `vite.config.js`:

```js
import { PubSub } from 'meteor/jam:pub-sub';

PubSub.configure({
  cache: true, 
  cacheDuration: 3 // I used 3 here just to make the testing easier, you can use whatever you think is appropriate for your app
});
```

Once you confirm the fix, I think we can close this issue. Thanks!

**Comment by harry-73** [comment]
Hi,

I confirm. It is solved.

Thanks.
